tests/manifests/test_tabular.py

```diff
@@ -216,6 +216,16 @@ def test_prop_type_not_given(tmp_path, rc):
     )
 
 
+def test_prop_type_required(tmp_path, rc):
+    check(tmp_path, rc, '''
+    d | r | b | m | property | type
+    example                  |
+                             |
+      |   |   | City         |
+      |   |   |   | name     | string required
```

Comment:
```suggestion
      |   |   |   | name     | string required
      |   |   |   | place    | geometry(point) required
```

I think, this addition will not work.